reaktive/src/jvmJsCommonMain/kotlin/com/badoo/reaktive/observable/plugins/ObservablePlugins.kt

```diff
@@ -0,0 +1,11 @@
+package com.badoo.reaktive.observable.plugins
+
+import com.badoo.reaktive.annotations.ExperimentalReaktiveApi
+import kotlin.jvm.JvmField
+
+@ExperimentalReaktiveApi
+actual object ObservablePlugins {
```

Comment:
Implement also Maybe/Completable/Single?

Reply:
Sure, right after we agree on the API